src/translations/fr/common.json

```
@@ -0,0 +1,4 @@
+{
+  "success": "Votre action s'est réalisée avec succès !",
+  "error": "Oulaa, il y a une erreur avec cette action !"
```

```suggestion
  "error": "Oulaa, il y a eu une erreur avec cette action !"
```
C'est mieux avec ça je pense